src/requirements.txt

```diff
@@ -1,58 +1,58 @@
 -i https://pypi.python.org/simple
 annotated-types==0.6.0; python_version >= '3.8'
 asgiref==3.7.2; python_version >= '3.7'
-boto3==1.28.79; python_version >= '3.7'
-botocore==1.31.79; python_version >= '3.7'
+boto3==1.33.7; python_version >= '3.7'
+botocore==1.33.7; python_version >= '3.7'
 cachetools==5.3.2; python_version >= '3.7'
-certifi==2023.7.22; python_version >= '3.6'
+certifi==2023.11.17; python_version >= '3.6'
 cfenv==0.5.3
 cffi==1.16.0; python_version >= '3.8'
 charset-normalizer==3.3.2; python_full_version >= '3.7.0'
-cryptography==41.0.6; python_version >= '3.7'
+cryptography==41.0.7; python_version >= '3.7'
 defusedxml==0.7.1; python_version >= '2.7' and python_version not in '3.0, 3.1, 3.2, 3.3, 3.4'
 dj-database-url==2.1.0
 dj-email-url==1.0.6
-django==4.2.7; python_version >= '3.8'
+django==5.0; python_version >= '3.10'
 django-allow-cidr==0.7.1
 django-auditlog==2.3.0; python_version >= '3.7'
-django-cache-url==3.4.4
-django-cors-headers==4.3.0; python_version >= '3.8'
+django-cache-url==3.4.5
+django-cors-headers==4.3.1; python_version >= '3.8'
 django-csp==3.7
 django-fsm==2.8.1
 django-login-required-middleware==0.9.0
 django-phonenumber-field[phonenumberslite]==7.2.0; python_version >= '3.8'
 django-widget-tweaks==1.5.0; python_version >= '3.8'
 environs[django]==9.5.0; python_version >= '3.6'
-faker==19.13.0; python_version >= '3.8'
+faker==20.1.0; python_version >= '3.8'
 fred-epplib@ git+https://github.com/cisagov/epplib.git@d56d183f1664f34c40ca9716a3a9a345f0ef561c
 furl==2.1.3
 future==0.18.3; python_version >= '2.6' and python_version not in '3.0, 3.1, 3.2, 3.3'
 gevent==23.9.1; python_version >= '3.8'
-geventconnpool@ git+https://github.com/rasky/geventconnpool.git@1bbb93a714a331a069adf27265fe582d9ba7ecd4
+geventconnpool@ git+https://github.com/rasky/geventconnpool.git
```

Comment:
(Blocking) This will break the connection pool - it requires a specific commit number (of the latest commit). This library uses an old tag, so when you call this directly it grabs an old commit from 2014. Thats interesting that a pip update will do that, maybe we need to discuss this in the next dev huddle?

This is one downside of that library but easily mitigated by forking it

Reply:
Do we have that documented somewhere? This then breaks our current setup with updating python requirements to this will need to be added into the docs.

Reply:
Documented this in our files for now, I think the Pipfile will need to be updated to reflect this somehow so it doesn't keep changing, but I don't want to hold this one up longer